build.gradle

```diff
@@ -80,4 +82,21 @@ subprojects {
 
 task clean(type: Delete) {
     delete rootProject.buildDir
-}
\ No newline at end of file
+}
+
+release {
+    buildTasks = ['doRelease']
+    // Do not change the tagTemplate value to double quotes or the version will be evaluated before
+    // the SNAPSHOT version is updated to the release version.
+    tagTemplate = 'v$version'
+}
+
+// We aren't building or doing anything interesting for release.
+// We just update the version and generate the tag as CI will handle deployment.
+task doRelease {
```

Comment:
Yes. I'm distracted with some other stuff but will revisit this afternoon.

On Mon, Jan 11, 2016 at 9:27 AM, Elliotte Rusty Harold <
notifications@github.com> wrote:

> In build.gradle
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/341#discussion_r49325854>
> :
>
> > \ No newline at end of file
> > +}
> > +
> > +release {
> > +    buildTasks = ['doRelease']
> > +    // Do not change the tagTemplate value to double quotes or the version will be evaluated before
> > +    // the SNAPSHOT version is updated to the release version.
> > +    tagTemplate = 'v$version'
> > +}
> > +
> > +// We aren't building or doing anything interesting for release.
> > +// We just update the version and generate the tag as CI will handle deployment.
> > +task doRelease {
>
> @patflynn <https://github.com/patflynn> Do you want to do this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/341/files#r49325854>
> .
>